blurb/blurb.py

```diff
@@ -956,9 +956,13 @@ def init_tmp_with_template():
             failure = str(e)
 
         if not failure:
-            assert len(blurb) # if parse_blurb succeeds, we should always have a body
-            if len(blurb) > 1:
-                failure = "Too many entries!  Don't specify '..' on a line by itself."
+            with open(tmp_path, "rt", encoding="utf-8") as file:
+                for line in file:
+                    if line.startswith(".. gh-issue:"):
+                        issue_number = line.split(":")[1].strip()
+                        if issue_number.isdigit() and int(issue_number) < 32426:
```

Comment:
Should this also handle (and reject) non-numeric issue numbers?

Reply:
Hey @ezio-melotti, that's true. I'll split the two if statements to show different errors.